src/messaging/ExchangeContext.cpp

```diff
@@ -266,6 +267,30 @@ CHIP_ERROR ExchangeContext::SendMessage(Protocols::Id protocolId, uint8_t msgTyp
     }
 }
 
+CHIP_ERROR ExchangeContext::SendStatusReport(const Protocols::SecureChannel::StatusReport & statusReport)
+{
+    ChipLogProgress(ExchangeManager, "Sending status report. Protocol code %u, exchange " ChipLogFormatExchange,
+                    statusReport.GetProtocolCode(), ChipLogValueExchange(this));
+
+    auto handle = System::PacketBufferHandle::New(statusReport.Size());
+    VerifyOrReturnError(!handle.IsNull(), CHIP_ERROR_NO_MEMORY,
+                        ChipLogError(SecureChannel, "Failed to allocate status report message"));
+    Encoding::LittleEndian::PacketBufferWriter bbuf(std::move(handle));
+
+    statusReport.WriteToBuffer(bbuf);
+
+    System::PacketBufferHandle msg = bbuf.Finalize();
```

Comment:
Again, could just use `handle` here.  Or call the single handle `msg`, either way.